engines/dradis-api/app/controllers/dradis/ce/api/v1/attachments_controller.rb

```diff
@@ -0,0 +1,101 @@
+module Dradis::CE::API
+  module V1
+    class AttachmentsController < Dradis::CE::API::V1::ProjectScopedController
+      before_action :set_node
+
+      skip_before_action :json_required, :only => [:create]
+
+      def index
+        @attachments = @node.attachments.each(&:close)
+      end
+
+      def show
+        filename = params[:id]
+        begin
+          @attachment = Attachment.find(filename, conditions: { node_id: @node.id } )
+        rescue
+          raise ActiveRecord::RecordNotFound, "Couldn't find attachment with id '#{params[:id]}'"
```

Comment:
I wanted the error to be handled here:
https://github.com/dradis/dradis-ce/blob/master/engines/dradis-api/app/controllers/dradis/ce/api/api_controller.rb#L14
and render all errors from a single method.